src/scaife-widgets/state.js

```diff
@@ -0,0 +1,8 @@
+import { INITIAL_PASSAGE } from '../constants';
```

Comment:
I know `scaife-widgets` encapsulation is still WIP, but is there another way we can retrieve this value without depending on a "top-level" constants module?

Maybe as simple as `initialPassage` and `passage` within the scaife-widgets store as `null` and documenting at the Vue app level that a developer needs to populate those values (perhaps within `App.vue`)?

Reply:
And as I think about it further, this type of functionality might eventually move out to a library module.  Even for a "single-work" reader / ATLAS instance, that library module could handle querying ATLAS seeding the initial version / passage.